addons/gsri_opex/functions/fremm/fn_fremmAddSub.sqf

```diff
@@ -48,49 +57,22 @@ addMissionEventHandler ["Map", {
 	params ["_opened"];
 	// Detect if map is closed but do not check if the event is linked to this module
 	if!(_opened) then {
```

Comment:
If-then statements without an else nor continuation statements can be replaced by an exitWith :

```sqf
if (condition) then {
  [instructions]
}
```

versus

```sqf
if !(condition) exitWith {};
[instructions]
```